# SNPsInQTLSelection
This project is part of a project for a doctoral thesis. We developed a script for custom SNPs selection in QTL study using *vcf* files.

# Thesis abstract
The proton-pumping ATPase (H+-ATPase) is found in the plant plasma membrane and plays an essential role to generate an electrochemical proton gradient, which is important for nutrient uptake and intracellular pH regulation. Activation of H+-ATPase of the plasma membrane of Saccharomyces cerevisiae by glucose is supposedly attributed to an intracellular calcium signal correlated to phosphatidylinositol metabolism. The hypothesis that calcium signaling, when induced by glucose, occurs through the inositol 1,4,5-triphosphate (IP3) pathway that acts as a secondary messenger when induced by glucose has been tested. It was showed that in arg82Î” yeast strains, where there is no conversion of IP3 into IP4 and/or IP5, glucose-induced increase of IP3, H+-ATPase activation and calcium signaling are more pronounced. S. cerevisiae PJ694a strain with mutations in the ARG82 gene showed a higher activation of the H+-ATPase than the wild strain, indicating the possible relationship between the absence this protein and increasing of IP3 concentrations and cytosolic calcium with subsequently activation of the H+-ATPase. Previously, a phenotype difference was observed between the S. cerevisiae BY4742 and PJ694a strains through the H + -ATPase activity test, indicating that the H+-ATPase activation phenotype is related to several genes. In this context, the objective of this work was to use different genomic approaches to identify new components of glucose-induced regulation of the plasma membrane H + -ATPase activation. For this, the sequencing of the clustered segregating genome, QTL mapping and Bioinformatics analyzes were carried out to identify the possible reason for the phenotype difference and/or new components of this signaling transduction pathway. Once the variants were identified in the QTL mapping analysis, the SNPsInQTLselection.py script was developed to identify the genetic bases that may be involved with the phenotype of higher cytoplasmic membrane activity H+-ATPase. Through this script, 42 variants were identified in 33 genes potentially involved with the phenotype of interest. To prioritize these candidate genes, the enrichment and interatoma analysis were performed, which allowed the identification of 10 enriched genes involved in the telomerase and phosphatidylinositol pathway (STT4, PIK2, UGA2, EST1, MEC3, HEK2, TOP3, PSO2, STO1, FUR4). Background BY strains containing these respective deleted genes (EUROSCARF collection) were used to test the extracellular acidification phenotype and glucose-induced calcium signaling. Through these tests, 4 (UGA2, FUR4, EST1 and STT4) of the 10 enriched genes showed the phenotype of greater activation of H+-ATPase, a phenotype that was evidenced in the extracellular acidification and calcium signal tests. Of these 4 genes, it is worth highlighting STT4 since, according to the function described for this gene in the literature, phosphatidylinositol-4-phosphate can play a role in regulating the activation pathway of H+-ATPase, controlling the activity of phospholipase C. In view of a positive relationship between H + -ATPase activity and fermentative performance, future studies can be done with the genes / alleles identified in this work, with application in industrial yeast strains to improve the efficiency of fermentation processes. The genes (UGA2, FUR4, EST1 and STT4) deserve detailed investigation to verify their possible involvement in the H+-ATPase signaling pathway. This work presents for the first time a Bioinformatics strategy to identify candidate genes when the QTL mapping has parental strains with high homozygosity.
